web/index.pug

```diff
@@ -7,6 +7,93 @@ block append variables
 
 block maincontent
 	.page.content
+		.art 
+			<svg width="100%" viewBox="0 0 1265 750" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
this is a large svg file.

- can we place this svg into a new file `web/assets/components-illustration.svg`
- then, can we find a good way in pug to reference the file, instead of placing the whole thing inline?

Reply:
sure.